test_project/test_utils/dbt_test_utils.py

```diff
@@ -154,6 +167,19 @@ def run_dbt_seed(self, seed_file_name=None) -> str:
 
         return self.run_dbt_command(command)
 
+    def run_dbt_seed_model(self, seed_model_name=None) -> str:
```

Comment:
This already exists as _**run_dbt_seed**_ above this function. What was the reason for creating this variation?

Reply:
The preceding function run_dbt_seed() runs a seed file, this function runs a seed model, i.e. a model being used as a seed file, to get around MSSQL issues elsewhere.

Reply:
This ties in with the new context_table_to_model() function below. With the luxury of a lot more time it should be possible to find a less workaround-y solution, but for now I focussed on finding a reasonable solution which would work for both default/Snowflake and MSSQL.